CondFormats/CSCObjects/interface/CSCDBL1TPParametersExtended.h

```diff
@@ -0,0 +1,133 @@
+#ifndef CSCObjects_CSCDBL1TPParametersExtended_h
+#define CSCObjects_CSCDBL1TPParametersExtended_h
+
+#include "CondFormats/Serialization/interface/Serializable.h"
+
+#include <vector>
+
+/** \class CSCDBL1TPParametersExtended
+ *  \author Sven Dildick
+ *
+ * Description: Configuration parameters needed for the Phase-1 upgrade of the level-1
+ *              CSC trigger primitives emulator. ME1/1 ALCT, CLCT and TMB have own members.
+ *
+ */
+
+union S{
+  unsigned int i;
+  bool b;
+};
+
+class CSCDBL1TPParametersExtended
+{
+ public:
+  CSCDBL1TPParametersExtended();
+  ~CSCDBL1TPParametersExtended();
+
+  int getValueInt(const std::string&) const;
+  bool getValueBool(const std::string&) const;
+
+  void setValue(const std::string&, int);
+  void setValue(const std::string&, bool);
+
+ private:
+  std::vector<int> paramsInt_;
+  std::vector<int> paramsBool_;
+
+  std::vector<std::string> paramNamesBool_{
```

Comment:
@dildick purely from this class, I don't see why these string vectors are not `const`